dotcom-rendering/package.json

```diff
@@ -27,7 +27,10 @@
 		"unused-exports": "yarn ts-unused-exports ./tsconfig.json --ignoreFiles='(/(fixtures|__mocks__)/|.+\\.(stories|mocks))' --exitWithCount",
 		"storybook": "start-storybook -p 4002",
 		"build-storybook": "node --max-old-space-size=4096 $(yarn bin)/build-storybook",
-		"makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/webpack/webpack.config.js"
+		"makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/webpack/webpack.config.js",
+		"cdk:build": "tsc -p ./tsconfig.cdk.json",
```

Comment:
Interesting naming read [here](https://stackoverflow.com/questions/45034124/what-naming-convention-do-i-use-for-scripts-in-package-json).

Reply:
Personally quite fond of using colons to group scripts, but I appreciate it if others arent.

We use use this same syntax for our other scripts such as cypress and jest.

Reply:
I'm also fond of the colon naming scheme